### ISSUE TITLE
overfitting issue

### ISSUE THREAD

**Issue by dichencd** [sshaoshuai/MTR#33]
Hi @sshaoshuai,

Thank you for sharing your great work. In your paper, you mentioned that you trained the model for 30epochs without data augmentation. I wonder whether you have encountered any overfitting issue (especially in terms of the validation loss)? How is the 30epoch selected? 

Thank you very much!

**Comment by sshaoshuai** [comment]
I didn't encounter overfitting issue with 30 epochs, but more epochs may lead to the overfitting problem. Actually I didn't conduct detailed ablation experiments about the number of epochs, so you may achieve better performance by using different numbers of epochs.

**Comment by dichencd** [comment]
I see. Thanks for your quick response! 
Meanwhile, I wonder whether there is a reason for choosing 30epochs? Thanks!

**Comment by sshaoshuai** [comment]
As I mentioned, I didn't conduct detailed ablation experiments about the number of epochs. I just tried several options and 30 epochs performed well on my side.

**Comment by dichencd** [comment]
I see. Thank you very much!
